runtime/doc/options.txt

```diff
@@ -2828,6 +2828,17 @@ A jump table for the options with a short description can be found at |Q_op|.
 	    :highlight Cursor gui=reverse guifg=NONE guibg=NONE
 	    :highlight Cursor gui=NONE guifg=bg guibg=fg
 <
+
+
+Terminal support (x => supported, empty => unsupported)
+                  |shape|color|transparency|reverse
+__________________|________________________________
+vte-based (> 3.9) |  x  |  x  |      x     |
+iterm             |  x  |  x  |            |
+Konsole           |  x  |     |            |
+Rxvt              |  x  |     |            |
```

Comment:
Would prefer not to have terminal minutiae in the docs. It could live in the FAQ on the wiki (and perhaps `man nvim`), but in general users should understand that Nvim does the best it can, and if it doesn't work in their terminal then it's just not supported.